### PR TITLE
5561 test configuration proxy

### PR DESCRIPTION
**Pull request by MatthieuMan**

### CODE REVIEW

**Comment by MatthieuMan** [comment]
changes done, correct?

File: behat.yml [hunk 1]
```diff
@@ -283,4 +283,8 @@ default:
 
     meta_service_dependency_configuration:
       paths: [ %paths.base%/features/MetaServiceDependencyConfiguration.feature ]
-      contexts: [ MetaServiceDependencyConfigurationContext ]
\ No newline at end of file
+      contexts: [ MetaServiceDependencyConfigurationContext ]
+    
+    test_proxy_configuration:
+      paths: [ %paths.base%/features/TestProxyConfiguration.feature ]
+      contexts: [ TestProxyConfigurationContext ]
```
Comment (ganoze) [inline behat.yml]: Missing \n.

File: features/bootstrap/TestProxyConfigurationContext.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        ));
+        $this->page->save();
+    }
+    
+    /**
+     * @Then a popin displays an error message
+     */
+    public function aPopinDisplaysAnErrorMessage()
+    {
+
```
Comment (ganoze) [inline features/bootstrap/TestProxyConfigurationContext.php]: Extra empty line.

**Comment by MatthieuMan** [comment]
changes done

File: behat.yml [hunk 3]
```diff
@@ -283,4 +283,9 @@ default:
...
+
```
Comment (ganoze) [inline behat.yml]: Useless whitespaces.

Comment (MatthieuMan) [reply behat.yml]: you did not ask for a Missing \n in the last comment?

**Pull request merged**